src/sentry/search/events/builder/metrics.py

```diff
@@ -965,6 +978,35 @@ def use_case_id_from_metrics_query(self, metrics_query: MetricsQuery) -> UseCase
 
         return use_case_ids.pop()
 
+    def resolve_ondemand_orderby(self) -> Any:
+        result = []
+        raw_orderby = self.raw_orderby
+
+        if not raw_orderby:
+            return []
+
+        if isinstance(self.raw_orderby, str):
+            raw_orderby = [self.raw_orderby]
```

Comment:
FYI no coverage for this block in case we want to address it in this PR:
<img width="372" alt="image" src="https://github.com/getsentry/sentry/assets/44410/91416331-1b41-42a2-b3b7-35173193ace1">

Reply:
Nah this is just typing cruft, pretty sure all the endpoints pass orderby as a string so it'd only happen if someone is calling the builder directly or from a new endpoint